web/modules/custom/itk_activity/src/Form/Multistep/MultistepFormImage.php

```diff
@@ -30,6 +31,11 @@ public function getFormId() {
   public function buildForm(array $form, FormStateInterface $form_state) {
     $form = parent::buildForm($form, $form_state);
 
+    // Set progress bar data.
```

Comment:
This seam to be the same 3 lines of code only the number change... shoul it have been in an setFromProgress() function.

Could 3 have been a name?